src/Microsoft.AspNet.Razor/TagHelpers/TagHelperAttributeDescriptor.cs

```diff
@@ -39,6 +40,8 @@ internal TagHelperAttributeDescriptor([NotNull] string name, [NotNull] PropertyI
         /// If <c>true</c> this <see cref="TagHelperAttributeDescriptor"/> is used for dictionary indexer assignments.
         /// Otherwise this <see cref="TagHelperAttributeDescriptor"/> is used for property assignment.
         /// </param>
+        /// <param name="usageDescriptor">The <see cref="TagHelperUsageDescriptor"/> that indicates how to use the
+        /// tag helper attribute during design time.</param>
```

Comment:
Wont add remarks but will change wording to do `contains information about use of this attribute` :+1: